### ISSUE TITLE
"lime create project" doesn't compile (missing Assets dir)

### ISSUE THREAD

**Issue by Gama11** [openfl/lime#975]
```bash
> lime create project
> cd SampleProject
> lime build neko
Error: Could not find asset path "Assets"
```

The created project doesn't have an `Assets` directory. There is an (empty) `Assets` directory in  `C:\HaxeToolkit\haxe\lib\lime\4,1,0\templates\project` however. Maybe empty directories are simply not copied?

```
lime 4.1.0
lime-samples 4.0.1
```

**Comment by jgranick** [comment]
Yes, I believe the "sparse copy" feature added at that time stopped copying empty directories. However, we added a `.gitignore` to prevent this problem, but is maybe a better solution anyway with version control

**Comment by Gama11** [comment]
That `.gitignore` is missing from releases though (including the latest 5.0.3) / it still doesn't work.

**Comment by Gama11** [comment]
Same issue with Lime 5.1.0. @jgranick This should probably be reopened?

**Comment by jgranick** [comment]
You're right, it looks like our build script strips it out. Either we should change the "copy template" logic to copy empty folders again, or add these files into the release archive

Thank you

**Comment by jgranick** [comment]
I've modified our build script, and re-released Lime 5.1.0 and OpenFL 5.1.3 with the additional hidden files
